dell/xps/15-9530/fingerprint/default.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs, lib, config, ... }: {
+
+  environment.systemPackages = [
+
+  ];
+
+  services.fprintd = {
+    enable = true;
+    tod = {
+      enable = true;
+      driver = pkgs.libfprint-2-tod1-goodix;
+    };
+
+  };
+
+  # Used to allow a password login on first login as an alternative to just a fingerprint
+  security.pam.services.login.fprintAuth = false;
+  # similarly to how other distributions handle the fingerprinting login
+  security.pam.services.gdm-fingerprint =
```

Comment:
```suggestion
  security.pam.services.gdm-fingerprint = lib.mkDefault
```

Reply:
Just by glancing at this, I cannot tell what it does, but it does not look hardware related.